dbus-wait.sh

```diff
@@ -2,13 +2,14 @@
 
 wait_for_dbus() {
 	while true; do
-		dbus_wait=$(dbus-send --system \
+		dbus-send --system \
```

Comment:
My bad, I didn't pay close enough attention to your final changes in #41. This looks correct now.